src/element.js

```diff
@@ -150,6 +150,7 @@ TreemapElement.defaults = {
   groupDividers: false,
   groupLabels: undefined,
   spacing: undefined,
+  formatter: undefined,
```

Comment:
I didn't do it before, because in this way, the display of the label is enabled for all and it's changing the current behavior. To have the current behavior, the user must set `formatter` to `undefined`.

Reply:
Let me try, I need more time than I expected ;) ! Maybe new PR is needed, closing this one.